test_rclcpp/CMakeLists.txt

```diff
@@ -104,6 +104,18 @@ if(BUILD_TESTING)
   # Macro for tests that trigger the shutdown of an executable based on particular console output,
   # then check the output of the executable against different console output.
   macro(custom_launch_test_executable_output test_name executable)
+    cmake_parse_arguments(_ARG "SKIP_TEST" "TIMEOUT" "" ${ARGN})
+    if(_ARG_UNPARSED_ARGUMENTS)
+      message(FATAL_ERROR "custom_launch_test_executable_output() called with "
+      "unused arguments: ${_ARG_UNPARSED_ARGUMENTS}")
+    endif()
+    if(_ARG_TIMEOUT)
+      set(_ARG_TIMEOUT "TIMEOUT" "${_ARG_TIMEOUT}")
+    endif()
+    if(_ARG_SKIP_TEST)
+      set(_ARG_SKIP_TEST "SKIP_TEST")
+    endif()
+
```

Comment:
Since this macro is only used for internal test I don't think this argument parsing is necessary.